docs/development/maintenance.md

```diff
@@ -13,104 +15,124 @@ modules of ILIAS.
+in our Community and contribute to the development of ILIAS and its code base,
+we split the code into units (often called components, even though the term
+is hard to define) and we define a set of authorities community members can have
+concerning these units of code. We understand an authority as the counterpart of
+a responsibility: The people having the authorities to do something in a unit of
+code also assume the responsibility for the corresponding functions.
+
+For the context of ILIAS, we define **four** different authorities:
+
+1. **Authority to Sign of Code Changes**
+2. **Authority to Sign of Conceptual Changes**
+3. **Authority to Sign of Testcase Changes**
+4. **Authority to (De-)Assign Authorities**
+
+Each of these authorities can be held by a different set of people. This means
+that the social organisation of different groups working on different parts of
+the Code of ILIAS can be different.
+Right now ILIAS knows a few different social structures for the maintenance of
+units in the code of ILIAS:
+
+* In the **"Classic Model"** all authorities are concentrated in one person and
+this person works mostly alone.
+* In the **Coordinator Model** all authorities are concentrated in one or more
+people and they work together with other developers in the community to improve
+the code.
+* In the **"Test and Assessment Model"** the authorities **to Sign of Conceptual
+Changes**, **to Sign of Testcase Changes**, and **to (De-)Assign Authorities**
+lies with one person and the **Authority to Sign of Code Changes** with two
+others.
+
+More will surely emerge as the optimal solution for each unit is found.
+
+# Responsibilites
+Independently of the social organisation for each unit of code the following
+responsibilites need to be assumed:
+
+* All people holding an authority must agree to coordinate the development
+of their unit of code with the product manager and with the people maintaining
+other units of code.
+* One of the people holding either the **Authority to Sign of Code Changes** or
+the **Authority to Sign of Conceptual Changes** gets assigned related bugs
+automatically by the [Issue-Tracker](https://mantis.ilias.de). S/he is responsible
+to make sure all issues receive a response within the defined time frame and are
+either fixed in a timely manner or postponed/closed with a solid explanation.
+* The people holding the **Authority to Sign of Code Changes** are responsible
+for Pull Requests to their component and get assigned related Pull Requests
+by the Technical Board according to the [Rules for Maintainers and Coordinators
+assigned to PRs](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/contributing.md#rules-for-maintainers-assigned-to-prs).
+* The person/people holding the **Authority to (De-)Assign Authorities**
+coordinate assignments of authorities with the Product Manager and the Technical
+Board, who hold a vetoing power over these decisions.
```

Comment:
Spitzfindigkeit: hier ist die Nein-/Ablehnungs-Seite beschrieben, aber nicht (exakt), wer die tatsächliche Eintragung macht. Wenn also der Schritt „coordinate assignment“ zwischen (De-)Assigning Authority und Product Manager+TB erledigt ist: soll dann die Person mit Authority to (De-)Assign Authorities die Änderung committen oder der Produktmanager oder jemand vom TB oder egal? (In der Praxis ist eine Klärung vielleicht übertrieben; könnte allerdings auch Klarheit schaffen, wenn wir möchten, dass jemand das einfach mal tut und das TB ggf. noch interveniert und „Veto!“ ruft.)

Reply:
Würde vorschlagen, dass wir das dann durch die Praxis regeln. Das kann ja sehr unterschiedlich sein, z.B. je nach dem, was diese Person für einen Hintergrund hat. "Coordinate" heisst meiner Meinung nach, dass wir miteinander sprechen müssen, bevor etwas passiert, sonst brauchen wir ein anderes Wording. Ich bin da völlig offen.